static/js/services/delete-docs.js

```diff
@@ -75,22 +108,33 @@ var _ = require('underscore');
         docs.forEach(function(doc) {
           doc._deleted = true;
```

Comment:
If we're going to push these ids remotely, why are we marking these as deleted?

Unless I'm misunderstanding, I imagine that at the start of this function you should branch based on offline/online, and only do this if the user is offline.

Reply:
Yeah you're right I'm probably branching a little bit too late into the function, I'll make it more explicit from the beginning which code is offline/online.